btk/obs_conditions.py

```diff
@@ -3,18 +3,76 @@
 import numpy as np
 import galsim
 import descwl
+from collections import namedtuple
 
+# A simple class archetype to serve as a dictionary without having to write the field names every time
+Survey = namedtuple("Survey", ["name", "pixel_scale", "psf_scale", "bands", "mean_sky_level", "exp_time", "zero_point"])
 
-all_surveys = {
-    "LSST": {
-        "name": "LSST",
-        "bands": ("y", "z", "i", "r", "g", "u"),
-        "pixel_scale": 0.2,
-    },
-    "DES": {"name": "DES", "bands": ("i", "r", "g", "z"), "pixel_scale": 0.263},
-    "CFHT": {"name": "CFHT", "bands": ("i", "r"), "pixel_scale": 0.185},
-    "HSC": {"name": "HSC", "bands": ("y", "z", "i", "r", "g"), "pixel_scale": 0.17},
-}
+pix_ROMAN = 0.11
+pix_RUBIN = 0.2
+pix_HST = 0.06
+pix_EUCLID = 0.101
+pix_HSC = 0.167
+pix_DES = 0.263
+pix_CFHT = 0.185
+
+# Sigma of the psf profile in arcseconds.
+sigma_ROMAN = 0.11 * np.array([1.68, 1.69, 1.86, 2.12, 2.44, 2.71])  # https://arxiv.org/pdf/1702.01747.pdf Z-band
+sigma_RUBIN = np.array([0.327, 0.31, 0.297, 0.285, 0.276, 0.267])  # https://www.lsst.org/about/camera/features
+sigma_EUCLID = np.array([0.16])  # https://sci.esa.int/documents/33859/36320/1567253682555-Euclid_presentation_Paris_1Dec2009.pdf
+sigma_HST = np.array([0.074])  # Source https://hst-docs.stsci.edu/display/WFC3IHB/6.6+UVIS+Optical+Performance#id-6.6UVISOpticalPerformance-6.6.1 800nm
+sigma_HSC = np.array([0.306, 0.285, 0.238, 0.268, 0.272])  # https://hsc-release.mtk.nao.ac.jp/doc/ deep+udeep
```

Comment:
Is there a reason for choosing numpy arrays here ?
Since these are constant values, we should rather use an immutable container like a tuple.
But since you're going for the `namedtuple`, which I think is a good idea, why not also define a `Filter` or `Bandpass` the same way you define a `Survey`, with its associated `psf_scale`, `mean_sky_level`, `exp_time` and `zero_point` ? That way each survey would have a collection of Filters.

Reply:
There was in my other codes where I was doing the operations on the vectors using np functions, but that is a fair point here, probably not needed.

I like the Filter class. I can implement that, it would make things look nicer and it makes a lot of sense.

Reply:
That seems to me to be a good idea, I think it would fit better with the Cutout system.

Reply:
I'll do that this afternoon after my meetings are over.

Reply:
I'm rolling back on the `Filter` objects. We will have to make distinction between the `g` band from HSC and the `g` band from Rubin for instance and that will lead to the creation of a lot of objects. Maybe there is a right way to do this, but I suggest to put it in a different PR then.